### PR TITLE
Hello dear mary,

### PR DESCRIPTION
**Pull request by aliakbar0**
Please review my assignment. Thanks,

### CODE REVIEW

File: activities.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<html lang = "en">
+<head>
+<title>Pacific Trails :: Activities</title>
+<link rel ="stylesheet" href="pacific.css">
+<meta charset="utf-8">
+</head> 
+<body>
+<div id="wrapper">
+<header>
+<h1>Pacific Trails Resort Activities</h1>
```
Comment (mbMosman) [inline activities.html]: The `<h1>` should not change.  It is always just Pacific Trails Resort.

File: activities.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+</header>
+<nav>
+<a href ="index.html">Home</a>&nbsp;&nbsp;
+<a href ="yurts.html">Yurts</a>&nbsp;&nbsp;
+<a href ="activities.html">Activities</a>&nbsp;&nbsp;
+<a href ="reservations.html">Reservations</a>&nbsp;&nbsp;
+</nav>
+<main>    
+<h2>Activities at Pacific Trails Resort</h2>
```
Comment (mbMosman) [inline activities.html]: This should be "Activities at Pacific Trails"

File: pacific.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
+body{
+    color: #666666;
+    background-color: #ffffff;
+    font-family: Arial,Helvetica Neue,Helvetica,sans-serif;
+    background-image: url(images/ptrbackground.jpg);}
+header {
+    color: #ffffff;
+    background-color: #000033;}
+h1 { background-color: #000033;
```
Comment (mbMosman) [inline pacific.css]: You want most of these styles to be in your header.  This is causing your header to be taller than needed.  

The height of 72px & line-height of 200% is not needed.  Move everything else but the margin and padding into the header and the size should adjust to something about half that size.

File: activities.html (hunk 1 above)
Comment (aliakbar0) [inline activities.html]: I was so involved in details that I forgot to review these issues.

File: yurts.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+<h1>Pacific Trails Resort</h1>
+</header>
+<nav>
+<a href ="index.html">Home</a>&nbsp;&nbsp;
+<a href ="yurts.html">Yurts</a>&nbsp;&nbsp;
+<a href ="activities.html">Activities</a>&nbsp;&nbsp;
+<a href ="reservations.html">Reservations</a>&nbsp;&nbsp;
+</nav>
+<main>
+<h2>The Yurts at Pacific Trails Resort</h2>
```
Comment (mbMosman) [inline yurts.html]: This should just be "The Yurts at Pacific Trails"

File: pacific.css [hunk 5]
```diff
@@ -0,0 +1,52 @@
+body{
+    color: #666666;
+    background-color: #ffffff;
+    font-family: Arial,Helvetica Neue,Helvetica,sans-serif;
```
Comment (mbMosman) [inline pacific.css]: Fonts with spaces in the name should have quotes around them, like "Helvetica Neue"

File: pacific.css [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    color: #ffffff;
+    background-color: #000033;}
+h1 { background-color: #000033; 
+     background-image: url(images/sunset.jpg); 
+     background-position: right; 
+     background-repeat: no-repeat; 
+     padding: 10px; 
+     height: 72px; 
+     color: #FFFFFF; 
+     font-family: Georgia, serif;
```
Comment (mbMosman) [inline pacific.css]: Additional backup fonts should be provided using a site such as CSS Font Stack or a similar reference.
`font-family: Georgia, Times, "Times New Roman", serif`

**Comment by mbMosman** [comment]
Good job! A few small issues to clean up, but it looks like you've got the hang of it. Make sure to take a last look at each page and compare to the wireframes before submitting.

**Pull request closed without merging**